srcpkgs/fatsort/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'fatsort'
+pkgname=fatsort
+version=1.6.2.605
+revision=1
+build_style=gnu-makefile
+hostmakedepends="help2man"
+short_desc="Utility that sorts FAT12, FAT16, FAT32 and exFAT partitions"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
+license="GPL-2.0-or-later"
+homepage="https://fatsort.sourceforge.io/"
+distfiles="https://iweb.dl.sourceforge.net/project/fatsort/fatsort-${version}.tar.xz"
+checksum=08cafa36f86ab89e7a3241d437df9af8a008549d7d13416256b7aaa006d5ffb7
+nocross=yes
+
+pre_install() {
+	sed -i 's|/usr/local/share|/usr/share|g' ${wrksrc}/Makefile
+	sed -i 's|/usr/local/sbin|/usr/bin|g' ${wrksrc}/src/Makefile
```

Comment:
Replace with
```
post_patch() {
	vsed -e 's|/usr/local|/usr|g' -i Makefile
	vsed -e 's|/usr/local/sbin|/usr/bin|' -i src/Makefile
}
```
Doing this in `post_patch` is logically consistent since it's a patch and will automatically place you in `$wrksrc` so the paths are simpler. Using `vsed` will verify that changes were actually made rather than silently proceeding if no matches are replaced.

Reply:
done